packages/fabric/tests/index.ts

```diff
@@ -23,6 +23,20 @@ test.serial("Include fabric middleware but do not use fabric statements", async
   t.is(res.status, 200, res.text);
 });
 
+test("Read stored data", async (t) => {
+  const { app } = t.context;
+  var res = await app.post("/fabric_retrieve");
+  t.is(res.status, 200, res.text);
+  t.is(res.body.hash, "Jmy2dTTA5QknOz+QL5HiH3iWI4eOuj2W9a6HkKdgdsI=");
+});
+
+test("Store data", async (t) => {
+  const { app } = t.context;
+  var res = await app.post("/fabric_store");
+  t.is(res.status, 200, res.text);
+  t.is(typeof res.body.hash, "string");
+});
+
 test("Missing step", async (t) => {
```

Comment:
Could you also please add a test in which there are many retrieve sentences? Does that case exists? Like a contract with:
```gherkin
Given I read from 'X' the data in tag 'aaaabbb999' and save the output into 'mimmo'
Given I read from 'Y' the data in tag 'cccccc' and save the output into 'mimmo'
```
or if can not be, then raise an error... even if I guess should be possible... anyway I need some testing cause will gonna change how the hooks works and don't want to break your code after the refactoring ;p

Reply:
Well for the moment only fabric is implemented with this kind of statement... So I cannot test two different statements with X different from Y...

I can surely test multiple statement which save data in different variables
```
Given I read from 'fabric' the data in tag 'aaaabbb999' and save the output into 'mimmo'
Given I read from 'fabric' the data in tag 'cccccc' and save the output into 'pippo'
```

What is the expected behaviour for saving multiple data in the same variable? I can save the names of all the tags and check that they are all different...

Reply:
Also actually in different variables ;) like 'mimmo' and 'pippo'